nss-day-cohort-43-csharp/MVC-Silver-Snakes
# Bs deactive user profiles

# Description
Created view and details for user profile and enabled deactivate user functionality.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Please ensure you can see a list of user profile and the details page for those profiles. Also please ensure you can deactivate users by clicking the Deactivate button beside users.
# Checklist:
